src/components/shared/thread/ActivityGraphFills.js

```diff
@@ -885,6 +849,8 @@ function _accumulateInBuffer(
   const intCategoryStartPixel = Math.floor(sampleCategoryStartPixel);
   const intCategoryEndPixel = Math.floor(sampleCategoryEndPixel);
   const intSamplePixel = Math.floor(samplePixel);
+  const prevInterval = sampleTime - prevSampleTime;
+  const nextInterval = nextSampleTime - sampleTime;
```

Comment:
Similarly here, I think `sampleTimeDeltaBefore` and `sampleTimeDeltaAfter` would be clearer names.